/**:
  ros__parameters:
    speed_to_erpm_gain: 4614.0   # ERPM/(m/s)
    speed_to_erpm_offset: 0.0    # ERPM
    steering_angle_to_servo_gain:  -1.2135  # ppm/rad
    steering_angle_to_servo_offset: 0.5304  # ppm
